app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -59,6 +64,10 @@ def connect(options = {})
     self.class.raw_connect(server, port, username, password)
   end
 
+  def vcd_service
+    @vcd_service ||= connect
+  end
```

Comment:
Don't need this method, can use either `with_provider_connection` or `ems.connect`